### ISSUE TITLE
As staff I can create a defaullt pet task with a 'due_in_days' value

### ISSUE THREAD

**Issue by kasugaijin** [rubyforgood/pet-rescue#403]
On hold. We should build out due dates on tasks, and recurring tasks, first. Then determine if this is required and how best to implement.

Blocked by this issue: https://github.com/rubyforgood/pet-rescue/issues/402

Default pet tasks allow staff to define a set of tasks that are common for each new pet. When a pet is created, the default tasks are used to create a set of new tasks for that pet via a service object.

A task may have a due date, and often these are a fixed time from when the task is created e.g., 14 or 30 days. So we want to enable staff to define due in days on a default pet task. For example, a default pet task with `due_in_days` of `10` will create a new task with a `due_date` of `Time.current + 10.days`. `due_in_date` is not required on a default pet task.

Acceptance Criteria
- add column to Default Pet Tasks table `due_in_days` type `integer`
- add the new field to Default Pet Tasks form
- modify service object to calculate the `due_date` for a task using `due_in_days`, otherwise result in `nil` if no `due_in_date`
- tests

**Comment by kasugaijin** [comment]
Replaced by https://github.com/rubyforgood/pet-rescue/issues/508
